study-builder/studies/pancan/emails/parental_welcome.en.html

```diff
@@ -0,0 +1,13 @@
+#set( $text = {
+"subject": "Thank you for submitting your information to Count Me In",
+"salutation": "Dear Parent,",
```

Comment:
Wondering if we can combine 2 emails (`parental_welcome.en.html`, `self_welcome.en.html`) into one because they are practically the same (except one word). Is it worth doing ? @MocanaAtBroad , @yufengwng 

see [similar Slack discussion](https://broadinstitute.slack.com/archives/G7ATA13D0/p1625768472269000)

Reply:
Yes, we should definitely try the approach mentioned in that Slack thread.

Reply:
done!